### ISSUE TITLE
Crash if Navigation.PopAsync in tap command on Android

### ISSUE THREAD

**Issue by ghost** [mrxten/XamEffects#7]
XAML
```
<Grid xe:TouchEffect.Color="Black"
	  xe:EffectsConfig.ChildrenInputTransparent="True"
	  xe:Commands.Tap="{Binding PageNavigationPopCommand}">
```

C#
```
PageNavigationPopCommand = new Command(async (dest) =>
{
	await Navigation.PopAsync(true);
});
```

If I write `await Task.Delay(500)` before PopAsync, there is no crash.

**Comment by mrxten** [comment]
Ah, it seems Control disposing before effect ends.
Thx, will fix it.

**Comment by Goncharuk-Nikita** [comment]
Yea, i have the same error:
![image](https://user-images.githubusercontent.com/30652592/39111521-1aeccc2c-46de-11e8-9768-52bf908fac3c.png)

**Comment by mrxten** [comment]
I know, havent time for fix it now.

**Comment by Goncharuk-Nikita** [comment]
Hi Vladislav! I'm waiting for your fix, and if you can not fix it now, can I do something?

**Comment by mrxten** [comment]
@FotalGoncharuk sorry, still too busy.
Yes, you can add `await Task.Delay(500);` before PopAsync,

**Comment by mrxten** [comment]
Fixed in 1.5.0
